### ISSUE TITLE
Allow for manual face tag

### ISSUE THREAD

**Issue by flammableliquids** [pulsejet/memories#733]
Sometimes recognise does not find a face. I would like to be able to manually tag a persons face.

**Comment by pulsejet** [comment]
Unlikely this can be implemented. The face needs to be associated with a vector, which we cannot calculate. This needs support from recognize.

**Comment by JeffAlyanak** [comment]
@pulsejet 

Wouldn't it be possible to just have a simple database table for manually tagged images that stores the X & Y of where the user clicks on the photo and the ID of the face/person?

This feature seems pretty useful, but I don't think it needs to be overly complicated. The UI could potentially even have a tooltip to differentiate manually-tagged people if you want to avoid users confusing them for automatically detected faces.

**Comment by mattdev44** [comment]
I agree with this feature request. I loaded ~13Gig worth of family pictures. On many pictures, Recognize only detected SOME of the faces. How can I tag the rest of the people manually, so that when I open "People" in the Memories app, the missing people can be tagged and show up?
